templates/patterns/social_media_links/pattern-social-media-links.html.twig

```diff
@@ -12,6 +12,7 @@
       path: link.url,
       label: link.label,
       icon_position: 'before',
+      hide_label: link.hide_label|default(false),
```

Comment:
Let's update one of the links in `social_media_links.ui_patterns.yml` so it's also visible in the patterns overview page.